gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +0,0 @@
-#Tue Jul 24 16:33:21 IST 2018
```

Comment:
This file was removed, I think this is making the build fail. Could you undo this removal?

Reply:
the change was still there, so I removed it for you :) Now you have to do `git pull` on your end :)